index.html

```diff
@@ -70,8 +60,8 @@ <h1 class="logo me-auto"><a href="index.html"><img src="assets/logo/logo-no-text
       <div class="row justify-content-center align-self-center" data-aos="fade-up">
         <div class="col-lg-6 intro-info order-lg-first order-last" data-aos="zoom-in" data-aos-delay="100">
           <h2>TechPals - Your software pals.</h2>
-          <h3>We can assist you building your next big thing by providing talented IT professionals</h3>
-          <h3>We comprise of great <strong><span>software engineers</span></strong>, <strong><span>agile coaches</span></strong> and <strong><span>product managers</span></strong> that will help you grow your product and build the next big thing.</h3>
+          <h3>Hire an agile software developer to help you build your next big thing.</h3>
```

Comment:
Why limit ourselves to developers?

```suggestion
          <h3>Hire an agile IT professional to help you grow your product and build the next big thing.</h3>
```